src/org/zaproxy/zap/extension/communityScripts/CommunityScriptFiles.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.communityScripts;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.channels.FileChannel;
+import java.util.Arrays;
+import java.util.Comparator;
+
+/**
+ * Simple utility class for listing the files to include in the ZapAddOn.xml file
+ * @author simon
+ *
+ */
+public class CommunityScriptFiles {
+	
+	public static final String ORIG = "src/org/zaproxy/zap/extension/svndigger/orig/"; 
```

Comment:
ORIG is never used, that I could see. Plus it points to the directory of another extension.